dts/arm/nxp/nxp_imx6sx_m4.dtsi

```diff
@@ -29,9 +29,10 @@
 		};
 	};
 
-	tcml:memory@1fff8000 {
+	/* 1fff8000 bootcode is aliased at 0x00000000 */
+	tcml:memory@0 {
 		compatible = "nxp,imx-itcm";
-		reg = <0x1fff8000 DT_SIZE_K(32)>;
+		reg = <0x00000000 DT_SIZE_K(32)>;
```

Comment:
Let's done change it to zero address, with startrom relocation, the size of zephyr.bin will be 512M bytes.

Reply:
@jlh-makeen , revert this update?